web/impact/impact/graphql/query.py

```diff
@@ -34,7 +35,7 @@ def resolve_expert_profile(self, info, **kwargs):
         return expert
 
     def resolve_entrepreneur_profile(self, info, **kwargs):
-        user_id = kwargs.get('id')
+        user_id = kwargs.get('id') or info.context.user.id
```

Comment:
use the default option for kwargs.get:

`user_id = kwargs.get('id', info.context.user.id)`